intro-section.tex

```diff
@@ -3,152 +3,272 @@ \section{Cylc Introduction}
+hello\_world, and run the following commands:
 
 \begin{lstlisting}[language=bash]
-$ cd hello_world  # move to the suite directory
+$ cd hello_world
 $ touch rose-suite.conf  # create an empty rose-suite.conf file
 $ rose suite-run  # run the suite
 \end{lstlisting}
 
-If successful cylc will generate some output then a window will open. This is
-the cylc monitoring and control tool which displays the current status of
-your suite. In the cylc gui you should briefly see the task
-\lstinline{hello_world}
-displayed with a coloured square next to it. The colour of the square
-represents the state of the task
-(for example green means the task is running and gray that it has succeeded).
-Once the hello\_world task has succeeded, having no more tasks to run the suite
-will automatically shut down.
+If successful cylc will generate some output and open the cylc monitoring and
+control GUI, which displays the current status of your suite. You should
+briefly see the task \lstinline{hello_world} displayed with a coloured square
+that represents its state. For example green means the task is running and gray
+that it has succeeded.  Once the task has succeeded, the suite has no more
+tasks to run and will automatically shut down.
+
+The task job script and its standard out and standard error are captured to
+a standard log directory location:
+\begin{lstlisting}[language=bash]
+$ ls -1 $HOME/cylc-run/hello_world/log/job/1/hello/01/
+job
+job.out
+job.err
+job.status
+job-activity.log
+\end{lstlisting}
+
+\more{The directory path contains the suite name (hello\_world), log type
+    ({\em job} or {\em suite}), task cycle point (1 for a non-cycling
+    suite), task name (hello), and task submit number (01). This avoids
+    unmanagably large flat log directories, and job logs don't get
+overwritten even if tasks are resubmitted.}
+
+While a task is still visible in the GUI you can right-click on it to view its
+log files.  After that, look in its log directory,
+
+\begin{lstlisting}[language=bash]
+$ cd $HOME/cylc-run/hello_world/log/job/1/hello/01/
+$ cat job.out
+\end{lstlisting}
+
+or use the \lstinline{cylc cat-log} command (which knows the file's location),
 
+\begin{lstlisting}[language=bash]
+$ cylc cat-log --help  # see "cylc --help" for top level command help
+$ cylc cat-log --stdout hello_world hello.1
+...
+Hello World!
+...
+\end{lstlisting}
+
+or use the fantastic \lstinline{rose suite-log} web suite log file viewer.
+
+\begin{lstlisting}[language=bash]
+$ cd hello_world
+$ rose suite-log
+\end{lstlisting}
+
+\more{
+    The \lstinline{rose suite-log} viewer will also be migrated to the cylc
+    project soon.
+}
 
 \subsection{Tasks}
+\label{Tasks}
 
-To convert a script into a cylc task the script should be placed inside a
-`bin` directory alongside the suite.rc file in the suite directory.
+Any program or script can be used as a cylc task without modification so long as,
```

Comment:
The original sentence was wrong.  (a) you don't need to "convert" your existing scripts in any way in order to automate their use with cylc - important because it speaks to the simplicity (well, relatively speaking!) of the system; and (b) and they don't need to be put in the suite bin directory either - that is just one option.    Perhaps you're referring mainly to the next line, the two conditions for use (error status, and no detached internal processes).  I suppose that could be shifted.